cmake/modules/RootMacros.cmake

```diff
@@ -1095,11 +1106,11 @@ endfunction()
 #                                 LINKDEF LinkDef.h LinkDef2.h : linkdef files, default value is "LinkDef.h"
 #                                 DICTIONARY_OPTIONS option    : options passed to rootcling
 #                                 INSTALL_OPTIONS option       : options passed to install headers
-#                                 NO_MODULE                    : don't generate a C++ module for this package
+#                                 NO_CXXMODULE                 : don't generate a C++ module for this package
```

Comment:
Is this not a backward-incompatible change? Why do we need to rename the option?

Reply:
This was a purely C++ modules related flag. I doubt anybody outside of ROOT have used it.

Reply:
Ok, if @Axel-Naumann approves, it's fine with me.

Reply:
Actually, I prefer to have it in the same commit. It shows the use-case of this flag.